api/utils/aws/identifiers.go

```diff
@@ -84,6 +84,9 @@ func IsValidRegion(region string) error {
 	if matchRegion.MatchString(region) {
 		return nil
 	}
+	if matchGlobalRegion.MatchString(region) {
+		return nil
+	}
```

Comment:
I don't think we want to include global regions here. for example, `aws-global` should not be a valid region for aws discovery:
https://github.com/gravitational/teleport/blob/b587910e09a4683359deb64481ce2b524963b20e/api/types/matchers_aws.go#L129-L133
same applies to `GetKnownRegions`.


I believe global regions are only valid for endpoint generation, but they are not real regions. we can either filter them out or create a separate list for it.

Reply:
I've split those into two variables, so they are already available if we want to use them.